scipy/optimize/_linprog_rs.py

```diff
@@ -398,8 +402,9 @@ def _phase_two(c, A, x, b, maxiter, tol, maxupdate, mast, pivot, iteration=0,
     return x, b, status, iteration
 
 
-def _linprog_rs(c, c0, A, b, x0=None, callback=None, maxiter=5000, tol=1e-12,
-                maxupdate=10, mast=False, pivot="mrc", _T_o=[], disp=False,
+def _linprog_rs(c, c0, A, b, x0, callback, postsolve_args,
```

Comment:
You've removed the None form x0, does this imply that x0 will always be constructed earlier?

Reply:
`_linprog_rs` is only called from `linprog`, which does provide an `x0`.